libs/SmartSync/AndroidManifest.xml

```diff
@@ -5,5 +5,10 @@
     android:versionCode="63"
     android:versionName="7.1.0.dev">
 
-    <application />
+    <application>
+        <service android:name="com.salesforce.androidsdk.analytics.AnalyticsPublisherService"
```

Comment:
Disabling ailtn in this test suite (we should do the same in other test suites).

Reply:
Good idea. Could you add them as part of this PR?